targeteditor/src/main/scala/explore/targeteditor/CoordinatesForm.scala

```diff
@@ -79,26 +83,28 @@ object CoordinatesForm {
                       focus = true,
                       icon = Icon("search")
           ),
-          FormGroup(widths = FormWidths.Equal)(
+          <.div(
+            ExploreStyles.FlexContainer,
+            ExploreStyles.TargetRaDecMinWidth,
             FormInputEV(
-              width = Seven,
               id = "ra",
               value = stateView.zoom(State.raValue),
               format = RightAscension.fromStringHMS,
-              label = "RA"
+              label = "RA",
+              clazz = ExploreStyles.Grow(1) |+| ExploreStyles.TargetRaDecMinWidth
             ),
             FormInputEV(
-              width = Seven,
               id = "dec",
               value = stateView.zoom(State.decValue),
               format = Declination.fromStringSignedDMS,
-              label = "Dec"
+              label = "Dec",
+              clazz = ExploreStyles.Grow(1) |+| ExploreStyles.TargetRaDecMinWidth
             ),
-            FormButton(width = Two,
-                       size = Small,
-                       icon = true,
-                       label = "X",
-                       onClick = props.goToRaDec(Coordinates(state.raValue, state.decValue))
+            FormButton(
```

Comment:
I think you had a way to remove the label keeping the alignment correct

Reply:
I can hide the label. Even if hidden, it seems like we should have a better label than `X`. Do you have a suggestion?

Reply:
Yeah X is bad :D. it maybe `GoTo` but still hidden